pkg/controller/masteruserrecord/masteruserrecord_controller.go

```diff
@@ -124,6 +125,11 @@ func (r *ReconcileMasterUserRecord) Reconcile(request reconcile.Request) (reconc
 			return reconcile.Result{}, err
 		}
 	}
+	if err = r.ensureTierTemplatesLabel(logger, mur); err != nil {
+		logger.Error(err, "unable to check labels on MasterUserRecord")
```

Comment:
Let's use `r.wrapErrorWithStatusUpdate()` instead of just logging. So it will log the error and set the status.

Reply:
done in 145c0d3 (along with tests)